.docker-compose.datagov-theme.yaml

```diff
@@ -12,6 +12,7 @@ services:
     - "5000:5000"
     build:
       args:
+        CKAN_BRANCH: 2.8
```

Comment:
It failed locally. _Master_ is having a lot of changes. It's not stable anymore for all extensions.
If you build locally you will see the errors.
The errors in my local machine stops when I add this line and re-build. 
We need to make this work over CKAN 2.8, not on _master_.

Reply:
Since the master branch is building in CI, the cause must be some local changes you've made.
If there's some issue with a specific extension you're trying to add, then please open an issue about the errors you're seeing.
Saying "it's not building, I'm getting some errors, install locally and you'll see" is not a good enough explanation.
BTW master branch is using ckan-2.8.1 and not ckan master.

Reply:
Removed. If I have any error I will define an issue for it.